bindings/python/CHANGELOG.rst

```diff
@@ -1,6 +1,12 @@
 Changelog
 =========
 
+Changes in Version 1.1.0b0
+--------------------------
+
+- Add support for Azure and GCP KMS providers.
+- Bundle libmongocrypt 1.1.0-beta1 in release wheels.
```

Comment:
You'l need to update the release scrip for this to be true: https://github.com/mongodb/libmongocrypt/blob/68a55e519b52b34dcc3ec56f776765eca01367ea/bindings/python/release.sh#L16

Reply:
Done.